charts/netbird/templates/dashboard-deployment.yaml

```diff
@@ -65,7 +65,7 @@ spec:
           {{- end }}
           ports:
             - name: {{ .Values.dashboard.service.name }}
-              containerPort: {{ .Values.dashboard.service.port }}
+              containerPort: {{ .Values.dashboard.service.containerPort | default .Values.dashboard.service.port }}
```

Comment:
It would be better to define this in values.xxx.deployment.containerPort and define this values in values.yaml file, with the same default values than the service.port.

We can add a breaking changes notices in the release, just in case some one were using a different default value for service.port.